.husky/pre-push

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm run test-all
+npm run platform-dist
+npx pnpm install -r
+git add pnpm-lock.yaml
```

Comment:
If `pnpm install` makes changes to `pnpm-lock.yaml` when this hook runs, `pnpm-lock.yaml` is added to staged files for commit, but not included in the pushed changes. Is that intended?

Reply:
Right, it was supposed to be in `pre-commit`